packages/babel-preset-env/test/fixtures/preset-options/deno-1_0/output.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+var _selfBrandCheck = /*#__PURE__*/new WeakSet();
+// These syntaxes should be transpiled:
+// static {} / ??= / #self in / C?.#self
+// The class declaration and the static private property should not be transpiled
+class C {
+  static #self = (_selfBrandCheck.add(this), C);
+  static #_ = C.#self ?? (C.#self = _selfBrandCheck.has(C === null || C === void 0 ? void 0 : C.#self));
```

Comment:
I don't understand why this is, in the previous babel8 test, the optional chain was not transpiled, now it is transpiled, and the test also passes.😕